src/mod.ts

```diff
@@ -185,6 +185,8 @@ class PathToTarkov implements IPreSptLoadMod, IPostSptLoadMod {
 
     this.pathToTarkovController.tradersController.initTraders(this.config);
 
+	this.pathToTarkovController.fleaController.initFlea(this.config);
```

Comment:
the config is directly passed here, see my review comment. I can guide you through a correct implementation here.